cc3d/core/PySteppables.py

```diff
@@ -1935,7 +1940,10 @@ def move_cell(self, cell, shift_vector):
         # Now we will delete old pixels
         medium_cell = CompuCell.getMediumCell()
         for pixel in pixels_to_delete:
-            self.cell_field[pixel.x, pixel.y, pixel.z] = medium_cell
+			# Safe deletion. Don't delete the old pixel 
```

Comment:
Indents need reduced.

Reply:
Thanks @JulianoGianlupi  for looking into this. Let me see how  this behaves in the example that Jim mentioned.